OpenRA.Mods.Common/SpriteLoaders/ShpD2Loader.cs

```diff
@@ -21,7 +21,7 @@ public class ShpD2Loader : ISpriteLoader
 		[Flags] enum FormatFlags : int
 		{
 			PaletteTable = 1,
-			SkipFormat80 = 2,
+			SkipLCW = 2,
```

Comment:
`SkipLCWCompression` please, in the hopes that it would be slightly less confusing to everyone if a few months.